operator/v1/types_console.go

```diff
@@ -22,12 +22,26 @@ type ConsoleSpec struct {
 	// customization options to the web console.
 	// +optional
 	Customization ConsoleCustomization `json:"customization"`
+	// provides contains configuration for using a specific service provider.
+	Providers ConsoleProviders `json:"providers"`
 }
 
 type ConsoleStatus struct {
 	OperatorStatus `json:",inline"`
 }
 
+type ConsoleProviders struct {
+	// statusPage contains ID for statuspage.io page that provides status info about.
+	// +optional
+	StatusPage *StatusPageProvider `json:"statusPageIO,omitempty"`
```

Comment:
The name is "Statuspage" so I'd suggest

```suggestion
	Statuspage *StatuspageProvider `json:"statuspageIO,omitempty"`
```

Reply:
One day they are going to have a branding crisis.... Statuspage, StatusPage, Status Page.... 😏